### ISSUE TITLE
Escaped HTML errors

### ISSUE THREAD

**Issue by angelortega** [snarfed/bridgy-fed#53]
Trying to use bridgy-fed I get the following error returned:

```
400 Bad Request
The server could not comply with the request since it is either malformed or otherwise incorrect.

Couldn't find a <a href="http://microformats.org/wiki/representative-hcard-parsing">representative h-card</a> on https://triptico.com/

```
Where the greater-than, less-than and quote symbols are incorrectly escaped, so I see them verbatim instead of a clickable link.

**Comment by snarfed** [comment]
thanks for the report! hmm, you're absolutely right. will fix.

**Comment by snarfed** [comment]
...although i wonder if i should be returning a Content-Type text/html response to a webfinger request at all, or if the spec says to report errors some other way. will look.

**Comment by angelortega** [comment]
> Closed #53 via c055f3a.

Thank you.
